packages/babel-plugin-proposal-decorators/src/index.js

```diff
@@ -28,6 +21,8 @@ export default declare((api, options) => {
     if (typeof decoratorsBeforeExport !== "boolean") {
       throw new Error("'decoratorsBeforeExport' must be a boolean.");
     }
+  } else if (!legacy) {
```

Comment:
I don't think we need to the option to toggle between before and after no more.
I believe there were very strong blocking arguments towards the exports before.